backend/src/BuiltinCli/Libs/LanguageServerProtocol.fs

```diff
@@ -0,0 +1,66 @@
+module BuiltinCli.Libs.LanguageServerProtocol
+
+open System.IO
+open System.Threading.Tasks
+open FSharp.Control.Tasks
+
+open Prelude
+open LibExecution.RuntimeTypes
+
+module Builtin = LibExecution.Builtin
+open Builtin.Shortcuts
+
+let types : List<BuiltInType> = []
+let constants : List<BuiltInConstant> = []
+
+// TODO reimplement this in Darklang, without any custom built-in function. I had
+// some difficulties around this - I forget what - but this got the job done for now
+let fns : List<BuiltInFn> =
+  [ { name = fn [ "LanguageServerProtocol" ] "readNextMessage" 0
```

Comment:
Perhaps for now we should use some names like Private or Danger so that other don't use these.

Another idea I had is that perhaps builtins can only be called from Stdlib functions, which hides them entirely from userspace.

Reply:
I'm hoping to delete this builtin fn before merging - I think I can replace it with a `Builtin.Stdin.todo(...)...` that captures just:
```fsharp
let streamReader = new StreamReader(System.Console.OpenStandardInput(),System.Text.Encoding.UTF8)
let buffer = Array.create length ' '
streamReader.ReadBlock(buffer, 0, length) |> ignore<int>
System.String buffer
``` 

rather than be LSP-specific

Reply:
ended up leaving this for another day, so added a "Danger" module here